packages/contracts/src/plugins/governance/majority-voting/addresslist/build-metadata.json

```diff
@@ -1,12 +1,65 @@
 {
   "ui": "",
-  "change": "",
+  "change": "- The ability to create a proposal now depends on the membership status of the current instead of the snapshot block.",
   "pluginSetupABI": {
-    "prepareInstallation": [
-      "tuple(uint8 votingMode, uint64 supportThreshold, uint64 minParticipation, uint64 minDuration, uint256 minProposerVotingPower) votingSettings",
-      "address[] members"
-    ],
-    "prepareUpdate": [""],
-    "prepareUninstallation": [""]
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 2.",
```

Comment:
Not sure if we need this description. In the end, it should be virtually the same as the general plugin `description`

Reply:
See here: https://github.com/aragon/osx/pull/375/files#r1200096582